### ISSUE TITLE
Request for annotations

### ISSUE THREAD

**Issue by meilijunxi** [CVMI-Lab/CoDet#8]
For some reason I cannot generate the correct instances_train2017_seen_2.json and instances_val2017_all_2.json. Can you provide a download link? thank you for your help！

**Comment by machuofan** [comment]
We have uploaded the annotation files and you can now download them from [here](https://drive.google.com/drive/folders/1z1kq41QR5oGOyCwx3QPs8lfU0ZWXy4mA?usp=sharing).

**Comment by meilijunxi** [comment]
@machuofan thank you.I tried to reproduce the results of the paper, but failed.You used 8 GPUs, the total batch size is 16?I use two GPUs, how should I set them up to reproduce the results of the paper?

**Comment by machuofan** [comment]
As mentioned in the paper, "The batch size on a single GPU is set to 2/8 for COCO/LVIS detection data and 8/32 for COCO Caption/CC3M caption data. The ratio between the detection batch and caption batch is set to 1:1 during co-training.". Therefore, if you are training on detection data only, the total batch size should be 16 for 8 GPUs.

In case you are training on 2 GPUs, you may considering rasing up the batch size on each GPU or decreasing the learning rate accordingly. For example, if your total batch size is 4, then you have to decrease the learning rate by 16 / 4 = 4.

**Comment by meilijunxi** [comment]
oh, I see. It seems that I was confused by IMS_PER_BATCH: 16 in the configuration file before. Thanks for your suggestion. If I increase the batch size of the detection data set, does the batch size of the caption data set need to be adjusted simultaneously?

**Comment by machuofan** [comment]
Yes.
